docs/templates/index.md

```diff
@@ -69,14 +69,37 @@ coder templates create <template-name>
 > [examples/](https://github.com/coder/coder/tree/main/examples/templates)
 > directory in the repo.
 
-## Configure Max Workspace Autostop
+## Template policies (enterprise)
 
-To control cost, specify a maximum time to live flag for a template in hours or
-minutes.
+By default, developers have unlimited workspace usage and can use any template version. Template policies allows you to:
 
-```sh
-coder templates create my-template --default-ttl 4h
-```
+- Auto-stop workspaces after 3 hours of inactivity.
+- Require workspaces to be restarted frequently to ensure they use the most recent template version.
+- Disable workspace auto-start.
+
+### Stop policy
+
+Stop policies require workspaces be stopped (or restarted), regardless of activity:
+
+- Daily (Restart on next quiet hours)
+- Weekly (Restart on weekends)
+- Custom (Every 2 weeks)
```

Comment:
Wouldn't this be after 2 weeks, as in "after 2 weeks of being online"?

Reply:
This would actually always shut down the workspace weekly if it is still online. For example:

- User starts workspace on Monday, it shuts down Friday if the user had a browser tab online the whole time 
- User starts workspace on Wednesday, it shuts down Friday if the user had a browser tab online the whole time

Reply:
ah ok, is the rationale to create consistency for the operator?

Reply:
Yeah.+ a guarantee all workspaces are up to date on Monday, for example.

In reality, I assume most workspaces will be auto-stopped during these hours